src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -56,6 +56,20 @@
   </div>
 
   <ng-template #cd_utilities>
+    <ng-container *ngIf="router.url === '/dashboard'">
+      <li class="nav-item">
+        <div class="form-check form-switch cd-navbar no-hover d-flex align-items-center">
+          <input class="form-check-input mb-1 blue-color"
+                 type="checkbox"
+                 [checked]="dashboardVersion === 'v3'"
+                 (change)="toggleContent()">
+          <a class="form-check-label no-hover"
+             i18n>
+            Dashboard v3
```

Comment:
@epuertat pointed out to me that v3 is mostly an internal term which user might not be aware of, so something like `Try out new dashboard` maybe?

Reply:
It also depends on the default dashboard, if we set up the new dashboard as the default one, I would go with something like `Old Dashboard` for the button text, to toggle to the current version of the dashboard

Reply:
The toggle can be used to switch back and forth right? So when on v3 it can show `Old dashboad` but when on v2 it shows `New dashboard`? 

Alternate would be to put the toggles only in the mgr-module option and update the documentation saying that you can switch between dashboards from the mgr module option?

Reply:
> Alternate would be to put the toggles only in the mgr-module option and update the documentation saying that you can switch between dashboards from the mgr module option?

In the end I went with this approach, it seemed more fitting now that we are storing the value within the mgr module. I couldn't find were to include it in the documentation. Could you point me to it?
Thanks @nizamial09

Reply:
There is an [Overview of the Dashboard Landing Page](https://docs.ceph.com/en/quincy/mgr/dashboard/#overview-of-the-dashboard-landing-page), and that can be updated to add a section for the new landing page and point it to the blog you wrote? And you can include the command to toggle the landing page in this section itself.

Reply:
@nizamial09 rather than linking from Ceph docs to the Ceph blog (docs should be, after code, the source of truth), I'd add the screenshots to the Ceph Dashboard docs. The lack of pictures in the documentation of a 100% visual component as the Ceph Dashboard has been a long-standing issue. So this would be a magnificent opportunity to start fixing that. If you Google for "Ceph Dashboard" first screenshots are from: SUSE, Rook, Ubuntu and Red Hat, but none of them from the Ceph Docs.

Reply:
Yeah, that sounds good. This can be a starting pointing for it. @Pegonzal can you do that?
=

Reply:
also please update the PR description as well, with the command to toggle between dashboards.